nps-unit-sample.json

```diff
@@ -1,367 +1,367 @@
-                      "slope": 2,
-                      "pvalue": 1
-                    }
-                  ]
-                }
-              ],
-              "constantLines": [
-                {
-                  "label": "Natural Condition, Haziest Days",
-                  "value": 12
-                },
-                {
-                  "label": "Natural Condition, Clearest Days",
-                  "value": 5
-                }
-              ]
-            },
-            {
-              "name":"Visbility on Clearest Days",
-              "yAxisLabel": "Haze Index, dv",
-              "variableLines": [
-                {
-                  "label": "Clearest Days",
-                  "annualValues": [
-                    {
-                      "year": 2000,
-                      "value": 8.0
-                    },
-                    {
-                      "year": 2001,
-                      "value": 8.1
-                    },
-                    {
-                      "year": 2002,
-                      "value": 8.2
-                    },
-                    {
-                      "year": 2003,
-                      "value": 8.3
-                    },
-                    {
-                      "year": 2004,
-                      "value": 9.0
-                    },
-                    {
-                      "year": 2005,
-                      "value": 10.5
-                    }
-                  ]
-                }
-              ],
-              "trendLines": [
-                {
-                  "year": 2000,
-                  "lines": [
-                    {
-                      "label": "Theil trend line slope = -0.18 dv/yr significant trend",
-                      "slope": -0.18,
-                      "pvalue": 0.05
-                    },
-                    {
-                      "label": "Some other trend line",
-                      "slope": 2,
-                      "pvalue": 1
-                    }
-                  ]
-                },
-                {
-                  "year": 2001,
-                  "lines": [
-                    {
-                      "label": "Theil trend line slope = -0.5 dv/yr significant trend",
-                      "slope": -0.5,
-                      "pvalue": 0.2
-                    },
-                    {
-                      "label": "Some other trend line",
-                      "slope": 2,
-                      "pvalue": 1
-                    }
-                  ]
-                }
-              ],
-              "constantLines": [
-                {
-                  "label": "Natural Condition, Haziest Days",
-                  "value": 12
-                },
-                {
-                  "label": "Natural Condition, Clearest Days",
-                  "value": 5
-                }
-              ]
-            }
-          ],
-          "compositionCharts": [
-            {
```

Comment:
`compositionCharts` work nearly identically to `trendCharts` except they're drawn as a stacked bar vs a line.

Reply:
Okay. If something doesn't apply, it will be empty.